nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -259,21 +264,49 @@ in
   # interface
   options = {
     services.wordpress = mkOption {
-      type = types.attrsOf (types.submodule siteOpts);
-      default = {};
```

Comment:
oh, this is basically the line you probably should add again

Reply:
@eonpatapon I'm sorry I messed up the indenting in the other suggestion. Maybe you want to instead add it here to the diff gets smaller and maybe you also want to squash your commits now as the PR seems pretty ready to me.